packages/crypto/tests/wycheproof_secp256k1.rs

```diff
@@ -102,8 +102,10 @@ fn ecdsa_secp256k1_sha256() {
                     let signature = from_der(&der_signature).unwrap();
                     let valid = secp256k1_verify(&message_hash, &signature, &public_key).unwrap();
                     assert!(valid);
-                    if tc.comment != "k*G has a large x-coordinate" {
-                        test_secp256k1_recover_pubkey(&message_hash, &signature, &public_key);
+                    if tc.comment == "k*G has a large x-coordinate" {
+                        // this case is currently not supported for historic reasons
```

Comment:
i.e. backwards compatibility / consistency?

Reply:
I elaborated the comment now to the following. Hope this is a bit clearer.

```rust
                        // This case (recovery ID 2 and 3) was never supported in the implementation of
                        // secp256k1_recover_pubkey because the library we used at that time did not support it.
                        // If needed, we could enable it now in a consensus breaking change.
```

The likelyhood for this to happen is so low that even Ethereum only supports recovery ID 0 and 1, not 2 and 3.

We can change it, not problem. But that's a topic to explore on its own. In this PR I just want to make the current behaviour a bit more visible.